app/src/main/res/values/strings_settings.xml

```diff
@@ -35,81 +35,82 @@
+    <string name="error_shortcut_start">The Android system could not open the shortcut</string>
 
     <!-- Search -->
     <string name="search_icon_title">Search</string>
-    <string name="search_hint">Search applications</string>
+    <string name="search_hint">Search for apps</string>
 
     <!-- Other settings -->
     <string name="set_transparent_status_bar">Transparent status bar</string>
     <string name="set_transparent_status_bar_hint">When favorites are closed</string>
-    <string name="set_hide_menu_button">Hide the menu button</string>
-    <string name="set_hide_menu_button_help">It stays accessible using the launcher icon</string>
+    <string name="set_hide_menu_button">No 🔧 menu button</string>
+    <string name="set_hide_menu_button_help">Always accessible from the launcher app icon</string>
     <string name="set_icon_pack">Use an installed icon pack</string>
-    <string name="set_icon_pack_none">None (use the real icons)</string>
-    <string name="set_hide_app_names">Hide applications names</string>
-    <string name="set_hide_app_names_help">Folder names will still be displayed</string>
-    <string name="set_remove_padding">Remove padding when names are hidden</string>
-    <string name="set_remove_padding_help">May produce unexpected results with long folders names</string>
-    <string name="set_force_portrait">Force portrait mode</string>
-    <string name="set_force_portrait_help">The launcher will not rotate even if automatic rotation is enabled</string>
-    <string name="set_immersive_mode">Use immersive mode</string>
-    <string name="set_immersive_mode_help">Hide system bars (long press on home screen to force if needed)</string>
-    <string name="set_reverse_interface">Reverse interface</string>
-    <string name="set_reverse_interface_help">Display the favorites panel on bottom</string>
-    <string name="error_appfilter_not_found">The appfilter.xml file was not found in the selected icon pack! (%1$s)</string>
+    <string name="set_icon_pack_none">None (use default icons)</string>
+    <string name="set_hide_app_names">No app names</string>
+    <string name="set_hide_app_names_help">Folder names are still shown</string>
+    <string name="set_remove_padding">No padding when app names are hidden</string>
+    <string name="set_remove_padding_help">Can cause problems with long folders names</string>
+    <string name="set_force_portrait">Forced portrait-mode</string>
+    <string name="set_force_portrait_help">Prevents the launcher rotating even if auto-rotation is on</string>
+    <string name="set_immersive_mode">Immersive mode</string>
+    <string name="set_immersive_mode_help">No system bars. (Can be changed by long pressing the home screen.)</string>
+    <string name="set_reverse_interface">Reversed interface</string>
+    <string name="set_reverse_interface_help">Apps atop, and favorites below the home screen</string>

```

Comment:
Could use italics for "atop" and "below".
The idea here is to call the home screen the thing in between.

Reply:
Indeed, good idea!
Would "above" be possible instead of "atop"?
(I don't think the italics are needed.)

Reply:
```suggestion
    <string name="set_reverse_interface_help">Apps above, and favorites below the home screen</string>

```